packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -96,6 +105,156 @@ export async function generateArmTemplate(ctx: SolutionContext): Promise<Result<
   return ok(undefined); // Nothing to return when success
 }
 
+export async function deployArmTemplates(ctx: SolutionContext): Promise<Result<void, FxError>> {
+  const azureInfraDir = path.join(ctx.root, baseFolder);
+
+  // update parameters
+  const parameterTemplate = await fs.readFile(
+    path.join(azureInfraDir, parameterFolder, parameterTemplateFileName),
+    ConstantString.UTF8Encoding
+  );
+  const parameterJson = JSON.parse(expandParameterPlaceholders(ctx, parameterTemplate));
+  const parameterDefaultFilePath = path.join(
+    azureInfraDir,
+    parameterFolder,
+    parameterDefaultFileName
+  );
+  await fs.writeFile(parameterDefaultFilePath, parameterJson);
+  const resourceGroupName = parameterJson.parameters.resourceBaseName;
```

Comment:
resource group name is generated by solution in other step. 
![image](https://user-images.githubusercontent.com/16605901/127460835-dd12fc6c-49c6-4904-8ee7-9ddcd6de2317.png)

Reply:
Fixed.